### ISSUE TITLE
Add multiplexer UGen

### ISSUE THREAD

**Issue by sensestage** [sensestage/supercollider#5]

**Comment by sensestage** [comment]
Two options:

* make a UGen with inputs: analogPin and multiPlex channel
* or use one channel input, which calculates which analogPin and multiPlex channel it should be.

**Comment by giuliomoro** [comment]
@sensestage I see you have created the ugens. Is there any code example for them?

**Comment by sensestage** [comment]
No, I only created a skeleton for them, they are far from finished or tested. So hence no code examples yet.
